float64                     azimuth
float64                     power
audio_common_msgs/AudioData audio